src/registrar/context_processors.py

```diff
@@ -60,35 +60,42 @@ def add_has_profile_feature_flag_to_context(request):
 
 def portfolio_permissions(request):
     """Make portfolio permissions for the request user available in global context"""
-    context = {
+    default_context = {
```

Comment:
nitpick:  
Not sure about this name.  Usually if we add "default" to something, it implies we will see variations later on in the subroutine...might just be my brain looking at it funny, but maybe try renaming this to "portfolio_context"?  "default_context" just kindof threw me off a bit.